skt/__init__.py

```diff
@@ -464,15 +470,15 @@ def prepare(self, clean=True):
             logging.info("cleaning up tree: %s", args)
             subprocess.check_call(args)
 
-        shutil.copyfile(self.basecfg, "%s/.config" % self.path)
+        shutil.copyfile(self.basecfg, self.get_cfgpath())
```

Comment:
This doesn't seem to be related, could you please do it in a separate commit?

Reply:
It's related, but separated.